api/v1/views/identity_membership.py

```diff
@@ -89,16 +90,16 @@ def delete(self, request, provider_uuid,
             return failure_response(
                 status.HTTP_404_NOT_FOUND,
                 "Identity does not exist.")
-        if not identity.can_share(user):
+        if not identity.can_share(request.user):
```

Comment:
You guys are right. But if you see L28 and L55. so I believe we should do it consistently.